accounts/abi/bind/bind.go

```diff
@@ -220,31 +214,79 @@ func Bind(types []string, abis []string, bytecodes []string, fsigs []map[string]
 					normalized.Inputs[j].Name = fmt.Sprintf("%s%d", normalized.Inputs[j].Name, index)
 				}
 				if hasStruct(input.Type) {
-					bindStructType[lang](input.Type, structs)
+					bindStructType(input.Type, structs)
 				}
 			}
 			// Append the event to the accumulator list
 			events[original.Name] = &tmplEvent{Original: original, Normalized: normalized}
 		}
+		for _, original := range evmABI.Errors {
```

Comment:
Are these changes still relevant, since you now created a `BindV2` method?

Reply:
Definitely not.